### PR TITLE
Edit DG for v1.3

### PR DESCRIPTION
**Pull request by alson001**
Removed unnecessary files.

### CODE REVIEW

File: docs/images/LogicClassDiagram.png
Comment (SeeuSim) [inline docs/images/LogicClassDiagram.png:1]: what's with the CSS/styling for the `Storage` package group

**Review by SeeuSim — APPROVED** [review]
Other than the minor styling flaw, everything else looks good. @JamesLiuZX @filbertphang do add your detailed reviews as well

**Review by filbertphang — APPROVED** [review]
Looks great! UML diagrams look fine and the explanations of how the Logic component was implemented is clear and well-written. LGTM 👍

**Review by JamesLiuZX — COMMENTED** [review]
LGTM! The language in DG section is clear, intuitive, and does not use more words than need to. Good PR overall, and just remember to restyle the `Storage` in `LogicClassDiagram` if not already done.

**Pull request merged**